src/Shell/Task/AssetsTask.php

```diff
@@ -157,6 +181,55 @@ protected function _process($plugins, $copy = false)
         $this->out('Done');
     }
 
+    /**
+     * Remove folder/symlink.
+     *
+     * @param array $config Plugin config.
+     * @return void
+     */
+    protected function _remove($config)
+    {
+        if ($config['namespaced'] && !is_dir($config['destDir'])) {
+            $this->out(
+                $config['destDir'] . $config['link'] . ' does not exist',
+                1,
+                Shell::VERBOSE
+            );
```

Comment:
You could use `$this->verbose()` or `$this->getIo()->verbose()` to have shorter lines.